patch/savefile.txt

```diff
@@ -0,0 +1,202 @@
+PatchSavefile:
+	call CheckNotEmpty
+	call CheckSavefileIsValid
+	call PrintSavefileMessage
+	; set #address, VersionInfo
+	; getbyteinc #result, #address
+	; getbyte #temp, #address
+	; mulacum #temp, #result, 0x100
+	set #temp, CURRENT_BUILD
+	ifgt #build, 9999, .invalid_build
+	ifgt #build, #temp, .future_build
+	ifeq #build, #temp, .latest_build
+	push #temp
+	bufstring .savefile_build_string_1
+	set #1, #build
+	set #2, 4
+	call PrintWithLeadingZeros
+	bufstring .savefile_build_string_2
+	printbuf
+	menu #result, YesNoMenu
+	jumpnz #result, .refused
+	; call PokecenterCheck
```

Comment:
If you're not using this, delete it

Reply:
I'm hoping to see if I can use it while also working on newbox development, but if it's not feasible with the build history (very good chance it isn't) I'll delete it